fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfiguration.java

```diff
@@ -33,6 +33,7 @@
     public static final String PROPERTY_CHECK_REFERENCE_TYPES = "fhirServer/core/checkReferenceTypes";
     public static final String PROPERTY_CONDITIONAL_DELETE_MAX_NUMBER = "fhirServer/core/conditionalDeleteMaxNumber";
     public static final String PROPERTY_SERVER_REGISTRY_RESOURCE_PROVIDER_ENABLED = "fhirServer/core/serverRegistryResourceProviderEnabled";
+    public static final String PROPERTY_SERVER_RESOLVE_FUNCTION_ENABLED = "fhirServer/core/serverResolveFunctionEnabled";
```

Comment:
as paul mentioned, we need to document this on in the user's guide

Reply:
Users Guide updates are forthcoming.

Reply:
Updated FHIR Server User's Guide with information on the new configuration property.